unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/Tabs/FriendsTabView.cs

```diff
@@ -59,7 +63,11 @@ public override bool RemoveEntry(string userId)
     public override bool UpdateEntry(string userId, FriendEntryBase.Model model)
     {
         if (!base.UpdateEntry(userId, model))
+        {
+            if (creationQueue.ContainsKey(userId))
```

Comment:
This looks like its addressing the corner case of "`UpdateEntry` fails because the entry has not been added yet by the queue processor". Maybe we should add a comment here to clarify this, because is not very obvious.

Reply:
The check was there before, I think just as a safe check. Now it works as you say, I just update the model to be always on sync with the last data received. Gonna comment it!